modules/environment-roles/templates/shared_terraform_policy_4.json.tpl

```diff
@@ -83,7 +83,9 @@
         "arn:aws:iam::${account_id}:role/TDRConsignmentExportRole${environment}",
         "arn:aws:iam::${account_id}:policy/TDRConsignmentExportPolicy${environment}",
         "arn:aws:iam::${account_id}:policy/TDRCreateDbUsersPolicy${environment}",
-        "arn:aws:iam::${account_id}:role/TDRCreateDbUsersRole${environment}"
+        "arn:aws:iam::${account_id}:role/TDRCreateDbUsersRole${environment}",
+        "arn:aws:iam::${account_id}:policy/TDRConsignmentApiAllowIAMAuthPolicy${environment}"
+
```

Comment:
Extra line break

Reply:
I have honestly been trying with the line breaks :smile: I've removed this one.